multiversx_sdk_cli/args_validation.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from multiversx_sdk_cli.errors import InvalidArgumentsError
+
+
+def ensure_required_transaction_args_are_provided(args: Any):
```

Comment:
:bulb: Maybe we can use "validate" or "check" instead of "ensure"? Since the word "ensure" makes us think about _required_ arguments.

Reply:
done